.github/workflows/nightly.yml

```diff
@@ -95,6 +95,11 @@ jobs:
     with:
       ref: ${{needs.create-nightly-tag.outputs.tag}}
 
+  run-playwright-tests:
+    uses: ./.github/workflows/playwright.yml
+    with:
+      ref: ${{ github.ref_name }}
```

Comment:
Think this needs to be so it runs on the newly generated nightly tag
```
ref: ${{needs.create-nightly-tag.outputs.tag}}
```

Reply:
whoops, indeed. Fixed it 👍